src/App.vue

```diff
@@ -385,9 +385,9 @@ export default class App extends Mixins(
       this.fileAndPayInvalidNameRequestDialog
     )
   }
-  /** The Fee Summary filing text for firm businesses. */
+  /** The Fee Summary filing text for firm dissolutions. */
   get filingLabelText (): string {
-    return this.isTypeFirm ? 'Dissolution' : ''
+    return (this.isTypeFirm && this.isDissolutionFiling) ? 'Dissolution' : null
```

Comment:
No, this getter is used for ALL entity types (not just firms or dissolutions).

I was thinking you should add a comment above the return statement describing that the text override is only for firm dissolutions.